src/olympia/addons/templates/addons/impala/review_list_box.html

```diff
@@ -4,10 +4,7 @@
     {% set replies = get_replies(reviews) %}
     {% for review in reviews %}
       <div class="review">
-        <h3>
-          {% if review.title %}
-            <b>{{ review.title }}</b>
-          {% endif %}
+        <h3></h3>
```

Comment:
I don't really care, but do we need the `<h3>`?

Reply:
I wrote back in April and can't remember now.  Maybe it was a css rule thing I was too lazy to fix?  I'll double check.